### PR TITLE
Add www.cem.va.gov redirects and remove legacy otherDomainRedirects.json

### PR DESCRIPTION
**Pull request by kelsonic**
## Description
**Issue:** https://github.com/department-of-veterans-affairs/va.gov-team/issues/4645#issuecomment-572117465 (these are the redirects)

This PR implements www.cem.va.gov redirects. This is the correct implementation, as opposed to my last PR: https://github.com/department-of-veterans-affairs/vets-website/pull/11436

## Testing done
N/A

## Screenshots
N/A

## Acceptance criteria
- [x] Add www.cem.va.gov redirects
- [x] Remove legacy redirects JSON that's misleading

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/proxy-rewrite/redirects/disabilityRedirects.json [hunk 1]
```diff
@@ -1,4 +1,79 @@
...
+    "dest": "/burials-memorials/memorial-items/presidential-memorial-certificates/"
+  },
+  {
+    "domain": "www.cem.va.gov",
+    "src": "/cem/pre-need/index.asp",
+    "dest": "/burials-memorials/pre-need-eligibility/"
+  },
+  {
+    "domain": "www.cem.va.gov",
+    "src": "/CEM/pre-need/FAQ/",
```
Comment (kelsonic) [inline src/applications/proxy-rewrite/redirects/disabilityRedirects.json]: This is the only one I'm worried about due to capital case 😬

File: src/applications/proxy-rewrite/redirects/index.js [hunk 2]
```diff
@@ -1,7 +1,4 @@
 import environment from '../../../platform/utilities/environment';
-// These pages are old va.gov pages that we are redirecting to
-// pages that we control
-// import redirects from './otherDomainRedirects.json';
```
Comment (kelsonic) [inline src/applications/proxy-rewrite/redirects/index.js]: Deleting this as it isn't used anymore and is misleading 👍

File: src/applications/proxy-rewrite/redirects/disabilityRedirects.json (hunk 1 above)
Comment (ncksllvn) [reply src/applications/proxy-rewrite/redirects/disabilityRedirects.json]: It converts to lowercase before comparing with the window location 🙂

**Review by ncksllvn — APPROVED** [review]

File: src/applications/proxy-rewrite/redirects/disabilityRedirects.json (hunk 1 above)
Comment (kelsonic) [reply src/applications/proxy-rewrite/redirects/disabilityRedirects.json]: Wonderful, thank you for confirming! 💯

**Review by cvalarida — APPROVED** [review]

**Pull request merged**